content/docs/1.4.0/best-practices.md

```diff
@@ -94,7 +94,7 @@ If you're using `ext4` as the filesystem of the volume, we recommend adding a li
 
 ## Volume Maintenance
 
-We highly recommend using the built-in backup feature of Longhorn.
+Using Longhorn's built-in backup feature is highly recommended. You can save backups to an object store (such as S3 and Azure), an NFS server, or an SMB or CIFS server. Saving to an object store is preferable.
```

Comment:
> Saving to an object store is preferable.

It would be nice if you could comment on the advantages of using an object store.